### DISCUSSION TITLE
Coingecko and/or Coinmarketcap as a source

### DISCUSSION THREAD

**Discussion by Seb-x1** [achannarasappa/ticker#244] [Ideas]
Hi, I can't be the only one who wish to have this, but please allow to select Coingecko and/or Coinmarketcap as a source of data, without having to use an individual API key.
I searched for Coingecko and Coinmarketcap and nothing came up...but I do believe that this would make this app simpler, safer for users (mismanaged API keys is risky), so way more appealing (I know barely use it due to those design flaws).
Thanks for your time and consideration.

**Comment by ajgringo619** [comment]
Doesn't Yahoo already use CoinMarketCap for its crypto prices?

**Comment by achannarasappa** [comment]
Coingecko as a datasource is already supported. No API key required.
https://github.com/achannarasappa/ticker#data-sources--symbols

Use the `.CG` suffix to reference any crypto asset from Coingecko

**Reply by ajgringo619** [reply]
I'm testing this on my watch list. While it works (and thanks for this!), I can't get it to sort properly, not like the Yahoo! lists were.
